guides/common/modules/con_lifecycle-environments-and-environment-paths.adoc

```diff
@@ -0,0 +1,18 @@
+[id="lifecycle-environments-and-environment-paths_{context}"]
+= Lifecycle environments and environment paths
+
+Application lifecycles are divided into *lifecycle environments* which represent each stage of the application lifecycle.
```

Comment:
emphasis differs to https://github.com/theforeman/foreman-documentation/pull/3281/files#diff-903bfa2707444b27ffa55804d122134d3f6dd94452dfba76856a0cb1409bd458R4

Reply:
Dropped all emphases.